backend/data/src/main/kotlin/io/tolgee/model/enums/Scope.kt

```diff
@@ -0,0 +1,146 @@
+package io.tolgee.model.enums
+
+import com.fasterxml.jackson.annotation.JsonValue
+import io.tolgee.constants.Message
+import io.tolgee.exceptions.BadRequestException
+import io.tolgee.exceptions.NotFoundException
+
+enum class Scope(
+  @get:JsonValue
+  var value: String
+) {
+  // !!!!
+  // Don't change the order of the enum, it's stored as ORDINAL is ApiKey entity
```

Comment:
This sound unnecessarily fragile.. Wouldn't it be possible to store the value in db?